libs/langchain-community/src/embeddings/voyage.ts

```diff
@@ -53,6 +58,7 @@ export class VoyageEmbeddings
 
   headers?: Record<string, string>;
 
+  inputType: string;
```

Comment:
I think this needs a `?` to mark as optional?

Reply:
yes you absolutely right, I should have secured with   
inputType?: string; // Make inputType optional

In the VoyageEmbeddings Class